e2e-tests/trade-finance-manager/cypress/integration/journeys/case/underwriting/underwriting-page.spec.js

```diff
@@ -0,0 +1,114 @@
+import relative from '../../../relativeURL';
+import partials from '../../../partials';
+import pages from '../../../pages';
+import MOCK_DEAL_MIA from '../../../../fixtures/deal-MIA';
+import { UNDERWRITER_MANAGER_1, T1_USER_1, UNDERWRITER_1 } from '../../../../../../e2e-fixtures';
+import { MOCK_MAKER_TFM, ADMIN_LOGIN } from '../../../../fixtures/users-portal';
+
+context('Underwriting page', () => {
+  let dealId;
+  const dealFacilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_MIA, MOCK_MAKER_TFM)
+      .then((insertedDeal) => {
+        dealId = insertedDeal._id;
+
+        const { dealType, mockFacilities } = MOCK_DEAL_MIA;
+
+        cy.createFacilities(dealId, mockFacilities, MOCK_MAKER_TFM).then((createdFacilities) => {
+          dealFacilities.push(...createdFacilities);
+        });
+
+        cy.submitDeal(dealId, dealType);
+      });
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN_LOGIN);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, MOCK_MAKER_TFM);
+    });
+  });
+
+  it('It should have the correct headings', () => {
+    cy.login(UNDERWRITER_MANAGER_1);
+    cy.visit(relative(`/case/${dealId}/deal`));
+
+    // go to lead underwriter page
+    partials.caseSubNavigation.underwritingLink().click();
+    pages.underwritingPage.dealHeading().contains('Deal');
+    pages.underwritingPage.underwritingAccordion().contains('Lead underwriter');
+    pages.underwritingPage.underwritingAccordion().contains('Pricing and risk');
+    pages.underwritingPage.underwritingAccordion().contains('Underwriter Manager\'s decision');
+  });
+
+  it('It should show unassigned and no decision yet if logged in as non-underwriter manager', () => {
```

Comment:
you already have the `it` part
so it should read
`it('should show...`

Reply:
done

Reply:
I can't see this being fixed?